### PR TITLE
refactor: remove unused parameter and function, reorganize function

### PR DESCRIPTION
**Pull request by Bingjiling**
Issue #, if available:

Description of changes:

Checklist:

- [ ] Have you successfully deployed to an AWS account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Did you add new integration tests?
- [ ] Did you run integration tests with your changes?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: fwoa-tools/src/integration-tests/migrationIntegTests.test.ts [hunk 1]
```diff
@@ -6,10 +6,16 @@ import { exec } from 'child_process';
 import axios, { AxiosInstance } from 'axios';
 import * as dotenv from 'dotenv';
 import createBundle from '../createPatientPractitionerEncounterBundle.json';
-import { binaryObject, binaryResource, getFhirClient, getFhirClientSMART } from '../migrationUtils';
+import { getFhirClient, getFhirClientSMART } from '../migrationUtils';
 
 dotenv.config({ path: '.env' });
 
+const binaryResource: { resourceType: string; contentType: string } = {
```
Comment (Bingjiling) [inline fwoa-tools/src/integration-tests/migrationIntegTests.test.ts:13]: These two consts are only used in test.

File: fwoa-tools/src/migrationUtils.ts [hunk 2]
```diff
@@ -25,13 +24,11 @@ export const POLLING_TIME: number = 5000;
 export const MS_TO_HOURS: number = 60 * 60 * 1000;
 export const EXPORT_STATE_FILE_NAME: string = 'migrationExport_Output.json';
 
-const getAuthParameters: (requestAdditionalScopes?: boolean) => { PASSWORD: string; USERNAME: string } = (
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationUtils.ts:28]: requestAdditionalScopes is only used in integ test

File: fwoa-tools/src/migrationUtils.ts [hunk 3]
```diff
@@ -180,24 +172,6 @@ export const getFhirClientSMART: (requestAdditionalScopes?: boolean) => Promise<
   });
 };
 
-export async function getResource(
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationUtils.ts:183]: Function is not used

File: fwoa-tools/src/migrationUtils.ts [hunk 4]
```diff
@@ -251,28 +225,3 @@ export async function checkConfiguration(logs: WriteStream, fhirServerType?: Fhi
 
   logs.write(`${new Date().toISOString()}: Finished checking configuration\n`);
 }
-
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-export async function verifyResource(
```
Comment (Bingjiling) [inline fwoa-tools/src/migrationUtils.ts:256]: Only used in verify script. Moved to verify script.

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**